### ISSUE TITLE
Using libraries/frameworks (jquery/bootstrap) in portfolio design

### ISSUE THREAD

**Issue by enderwillsaveu** [den-materials/wdi-orientation#9]
I see many articles showing how resources like jQuery and bootstrap can be helpful in designing projects. Would job recruiters look down on entry level applicants using these resources to assist in creating their portfolios? Would it be better to stick to strict js and css?

some links on the subject: 
https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&cad=rja&uact=8&sqi=2&ved=0ahUKEwjlyKLNwtzUAhVp6YMKHQfEAdkQFggkMAA&url=https%3A%2F%2Fwww.quora.com%2FIs-it-frowned-upon-to-use-Bootstrap-for-all-my-projects-in-my-front-end-web-development-portfolio&usg=AFQjCNFO7lUbVrCK5kw4QXhGWIJ4qNvDHw

https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=5&cad=rja&uact=8&ved=0ahUKEwjU3KKVw9zUAhVL3IMKHaYtA9IQFghBMAQ&url=http%3A%2F%2Fcoderchronicles.org%2F2016%2F11%2F04%2Fbuilding-your-first-portfolio-website-these-are-some-tips-to-consider%2F&usg=AFQjCNHHw0wDqc9DlqO3z6HWcX4Z7QtWiQ

I talked to Connie about this but would appreciate your opinion on this as well

**Comment by ZebGirouard** [comment]
Hi Tyler,

So I would, in general, preach the intersection of two things when building out your portfolio:

1) Put your *best* foot forward
2) Hiring managers/interviewers will probably spend < 10 minutes looking at your stuff

So with that advice comes a couple corollaries:

1) Almost certainly use some sort of template for your portfolio.  There is no point spending 12 hours trying to make something look pretty, and then still being frustrated about the color scheme.  There are people who have spent that time, and offer their hard work for free or cheap.

2) Almost certainly *don't* use a template for your proudest work (the 3 or 4 projects in your portfolio, or the starred repos in your GitHub), it demonstrates that you are proud of someone else's work, and interviewers will definitely ask you about "how you built _____" if you go far enough in the process.  You want to say, "Well, I needed to use Angular for the front end and Node for the backend," instead of, "Well, I downloaded this from freetemplates.com."

Now for Bootstrap and jQuery.  Every modern dev shop uses some "efficiency libraries" like these.  There is just too much to do and too little time to build everything from scratch.  You shouldn't need to build your own refrigerator, and you shouldn't need to use Vanilla JS or plain CSS for *everything* either.

That being said, you want to avoid being seen as a "one trick pony", especially with jQuery and Bootstrap because they are frameworks famous for producing people that can not use JS or CSS at all, respectively.

So in conclusion, you should have at least one project you are proud of *with* Bootstrap, and one *without*.  Same with jQuery.  After that, it's up to you what you are comfortable with and want to explore.

Did that answer your question?
